test/distrib/ruby/run_distrib_test.sh

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -ex
+set -eux
```

Comment:
I'm not sure about the `-u`.
I rarely saw anyone use it and because of its rarity, it think it's going to take people by surprise and cause more trouble than what it's worth.